pkgs/development/tools/minipro/unstable.nix

```
@@ -0,0 +1,49 @@
+{ stdenv, installShellFiles, which, fetchgit, git, pkg-config, libusb1 }:
+stdenv.mkDerivation rec {
+  pname = "minipro";
+  version = "0.4dev";
+
+  src = fetchgit {
+    url = "https://gitlab.com/DavidGriffith/minipro.git";
+    rev = "896fe7d9d6f1e77931626d273e880856884ee6d8";
+    sha256 = "1b14kg6aj23a808lngap63qf92193rx7s7l94ymf9dkwqa1xb524";
+    leaveDotGit = true;
+  };
+
+  nativeBuildInputs = [
+    pkg-config
+    installShellFiles
+  ];
+
+  buildInputs = [
+    stdenv
+    which
+    libusb1
+    git
+  ];
+
+  preConfigure = ''
+    export PKG_CONFIG="${pkg-config}/bin/pkg-config";
+    substituteInPlace Makefile --replace '$(shell date "+%Y-%m-%d %H:%M:%S %z")' "1970-01-01 00:00:00 +0000"
+  '';
+
+  buildPhase = ''
```

Could be using `makeFlags` for this.
```suggestion
  makeFlags = [ "-e minipro" ];
```